src/pages/Environments.js

```diff
@@ -456,8 +456,9 @@ const Environments = () => {
   // created, workspace namespace (a.k.a billing project) value used to equal the google project.
   // Therefore we use google project if the namespace label is not defined.
   const renderWorkspaceForApps = app => {
-    const { appType, googleProject, labels: { saturnWorkspaceNamespace = googleProject, saturnWorkspaceName } } = app
-    return getWorkspaceCell(saturnWorkspaceNamespace, saturnWorkspaceName, appType, false)
+    const { appType, googleProject, cloudContext: { cloudResource }, labels: { saturnWorkspaceName } } = app
+    const resolvedSaturnWorkspaceName = googleProject ? googleProject : cloudResource
```

Comment:
we can get rid of `googleProject`...it's not returned from Leo API anymore...

I think we should still try to use `saturnWorkspaceNamespace` if it exists

Reply:
I agree with this ^